### DISCUSSION TITLE
Question!

### DISCUSSION THREAD

**Discussion by bakinazik** [robertodoering/harpy#871] [General]
The twitter api was restricted yesterday. Will harpy be affected by this in the future?

**Comment by robertodoering** [comment]
It seems like a few third party clients were targeted. The Twitter API (both v1 and v2) is still working as it did before..
So not sure what will happen, we can only wait and see :/

**Comment by dnestico** [comment]
Sadly Harpy's days might be numbered :(... That would be the last straw for me... https://9to5google.com/2023/01/14/twitter-third-party-app-intentional-block/

**Reply by robertodoering** [reply]
yikes
